message_definitions/v1.0/ras_a.xml

```diff
@@ -10,7 +10,140 @@
+        <param index="3" label="Approach vector initial point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the initial point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="4" label="Approach vector initial point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the initial point of the approach vector to the portal. NaN if unknown.</param>
+        <param index="5" label="Approach vector final point X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="6" label="Approach vector final point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX if unknown.</param>
+        <param index="7" label="Approach vector final point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the final point of the approach vector to the portal. NaN if unknown.</param>
+      </entry>
+      <entry value="40510" name="MAV_CMD_DO_EXPLORATION" hasLocation="true" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>
+          Start or continue the exploration task.
+          If starting a new exploration, but requiring to get through a portal first, params 4 or 5 to 7 should be set and different from the specified ignore values.
+          Continuing an exploration can be done without necessarily setting the ingress portal, unless the exploration requires the definition of more than the initial ingress portal.
+          If the passed exploration ID does not exist or is not listed as a valid exploration ID in the vehicle autonomy engine, then this command should be rejected.
+          COMMAND_INT should be used so to set the MAV_FRAME and consequently, frame and coordinates of the portal position (MAV_FRAME_GLOBAL means global coordinates are being used,
+          while MAV_FRAME_LOCAL_NED means local NED coordinates are being used).
+        </description>
+        <param index="1" label="Exploration task ID" minValue="0" maxValue="255" increment="1">Sets the ID of the exploration task to start. If the ID already exists, it resumes that (queued/listed) exploration task. Set to UINT8_MAX if one wants to resume the last exploration task.</param>
+        <param index="2" label="Time limit" units="s" minValue="0">Time limit to execute the exploration. Reaching this time triggers the behavior defined in param 4. Set to 0 when there is no time limit.</param>
+        <param index="3" label="Behavior after stopping" minValue="0" maxValue="2" increment="1">The behavior after stopping the exploration task. 0: Hold, 1: Land, 2: Return to position.</param>
```

Comment:
Should this be "after finishing"? I wouldn't expect this behavior to trigger if the exploration is halted by a mode switch to POSCTL, for example.